src/bundle/Resources/views/content/tabs/locations.html.twig

```diff
@@ -1,50 +1,52 @@
 {% trans_default_domain "locationview" %}
 {{ form_start(actionsForm, {'action': path('ez_hybrid_platform_ui_location_actions', {'contentId': content.id, 'redirectLocationId': location.id})}) }}
 
-<div class="ez-list-toolbar">
-    <h2 class="ez-list-toolbar-label">{{ 'locationview.locations.content.locations'|trans()|desc('Content locations') }}</h2>
-    <div class="container-buttons">
-        {{ form_widget(actionsForm.parentLocationId, {'attr': {'placeholder': 'Location ID'}}) }}
-        {{ form_widget(actionsForm.add, {'label': 'locationview.locations.add.location'|trans()|desc('Add Location'), 'attr': {'class': 'ez-button ez-button-secondary'}}) }}
-        {{ form_widget(actionsForm.delete, {'label': 'locationview.locations.delete.locations'|trans()|desc('Delete selected locations'), 'attr': {'class': 'ez-button ez-button-negative'}}) }}
+<ez-selection-activate-element>
+    <div class="ez-list-toolbar">
+        <h2 class="ez-list-toolbar-label">{{ 'locationview.locations.content.locations'|trans()|desc('Content locations') }}</h2>
+        <div class="container-buttons">
+            {{ form_widget(actionsForm.parentLocationId, {'attr': {'placeholder': 'Location ID'}}) }}
+            {{ form_widget(actionsForm.add, {'label': 'locationview.locations.add.location'|trans()|desc('Add Location'), 'attr': {'class': 'ez-button ez-button-secondary'}}) }}
+            {{ form_widget(actionsForm.delete, {'label': 'locationview.locations.delete.locations'|trans()|desc('Delete selected locations'), 'attr':  {'class': 'ez-button ez-button-negative ez-js-activable-element', 'disabled': true}}) }}
```

Comment:
this line and the previous one are way too long. It's a good occasion to wrap the code like:
```twig
{{
    form_widget(actionsForm.delete, {
        'label': 'locationview.locations.delete.locations'|trans()|desc('Delete selected locations'),
        'attr':  {'class': 'ez-button ez-button-negative ez-js-activable-element', 'disabled': true}
    })
}}
```